### ISSUE TITLE
Access to issues for automated processing

### ISSUE THREAD

**Issue by rockdaboot** [google/oss-fuzz#3027]
Some projects would like to have the issues in their own bug tracker, see #2985. But there may also be the need for e.g. custom filtering, generating chat messages (bridging), etc.

As I understood, the is currently no way to retrieve that information with an automated tool. Of course there are emails with the issue reports, but these are hard to parse - formats can change any time.

Having a way to retrieve the issues (API or file download) in a defined state, e.g. documented JSON format, would be awesome. That would solve a bunch of needs as people can write their own bridges into their world (bug trackers, messaging, ...).

To have start, there could be a similar way as downloading the corpora for a single fuzzer.  
Downloading latest corpora:
```
gsutil cp gs://${project}-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/${project}_${fuzzer}/latest.zip
```
So I think of something like this, just for issues.

**Comment by oliverchang** [comment]
Thanks for the FR. We have a project in the works that may help with this use case, but we don't have an ETA right now and it may come some time next year.

**Comment by jonathanmetzman** [comment]
The project in the work is OSV which was released: https://osv.dev/

Discussion for API can continue here: #3925
